src/app/common/templates/job-body.html

```diff
@@ -30,7 +37,7 @@
             <li class="job-description">{{ job.attributes.description }}</li>
         </ul>
         <ul ng-if="job.attributes.description_translation.from != job.attributes.description_translation.to">
-            <li class="job-description translated" ng-show="dt.job_item">
+            <li class="job-description" ng-show="dt.job_item">
```

Comment:
I'm not familiar with this code, but why is `translated` removed? (We do want the job description translated.)